packages/react-components/src/mail.ts

```diff
@@ -4,12 +4,12 @@ interface MailOptions {
 }
 
 export const createMailTo = (
-  email: string,
+  emails: string | string[],
   { subject, body }: MailOptions = {},
 ): string => {
-  const mailTo = new URL(
-    `mailto:${email.split('@').map(encodeURIComponent).join('@')}`,
-  );
+
+  const list = typeof emails === 'string' ? new Array(emails) : [...emails];
+  const mailTo = convertEmailListToMailToUrl(list);
```

Comment:
```suggestion
  const mailTo = `mailto:${list.map((email)=>email.split('@').map(encodeURIComponent).join('@')}).join(',')}`,
```

Reply:
I found it more readable and maintainable to have a function to this. I'm exporting it so it can be used elsewhere and/or for testing (if necessary).